.github/workflows/container-deploy.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           echo "TF_SERVING_HOST=${{ secrets.TF_SERVING_HOST }}" >> .env
           echo "INFLUXDB_HOST=${{ secrets.INFLUXDB_HOST }}" >> .env
           echo "INFLUXDB_PORT=8086" >> .env
-          echo "INFLUXDB_NAME=db0" >> .env
+          echo "INFLUXDB_NAME=${{ env == 'robotoff-org' && 'off_metrics' || 'off_net_metrics' }}" >> .env
```

Comment:
I'm not sure `env == ` will work here, but `matrix.env ==` should ?

Reply:
thanks !